# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1473625448 25200
#      Sun Sep 11 13:24:08 2016 -0700
# Node ID c61467fc0d3a34f638278bcb09065f0a3a8a8f36
# Parent  7eaa642810f76205918ce80f72c0a9efd2e61a22
Add missing newline in output of 'make check' (bug #49052)

* __run_test_suite__.m: Add missing newline after log file name is printed.

diff --git a/scripts/testfun/__run_test_suite__.m b/scripts/testfun/__run_test_suite__.m
--- a/scripts/testfun/__run_test_suite__.m
+++ b/scripts/testfun/__run_test_suite__.m
@@ -82,17 +82,17 @@ function [pass, fail, xfail, skip] = __r
       endif
       puts ("\n");
       printf ("See the file %s for additional details.\n", logfile);
       if (dxf > 0)
         puts ("\n");
         puts ("Items listed as XFAIL above are known bugs.\n");
         puts ("Bug report numbers for them may be found in the log file:\n");
         puts (logfile);
-        puts ("Please help improve Octave by contributing fixes for them.\n");
+        puts ("\nPlease help improve Octave by contributing fixes for them.\n");
       endif
       if (dsk > 0)
         puts ("\n");
         puts ("Tests are most often skipped because the features they require\n");
         puts ("have been disabled.  Features are most often disabled because\n");
         puts ("they require dependencies that were not present when Octave\n");
         puts ("was built.  The configure script should have printed a summary\n");
         puts ("at the end of its run indicating which dependencies were not found.\n");
